oteapi/models/triplestoreconfig.py

```diff
@@ -26,11 +32,19 @@ class TripleStoreConfig(AttrDict):
         ...,
         description="AllegroGraph port number.",
     )
-    agraphUser: str = Field(
-        ...,
-        description="AllegroGraph user name.",
-    )
-    agraphPassword: SecretStr = Field(
-        ...,
-        description="AllegroGraph user password.",
-    )
+
+    @root_validator
+    def ensure_user_pass(cls, values: "Dict[str, Any]") -> "Dict[str, Any]":
+        """Ensure that user/password are set, since they are optional in the SecretConfig."""
+        if not all(values.get(_) for _ in ["user", "password"]):
+            raise ValueError("User and password must be defined.")
+        return values
+
+    class Config:
+        """Pydantic configuration for TripleStoreConfig."""
```

Comment:
Please add a one or two liner to the doc-string on what `fields` here is doing.

Reply:
Done 👍